src/sentry/api/serializers/models/actor.py

```diff
@@ -1,10 +1,11 @@
 from sentry.api.serializers import Serializer
 from sentry.models import Team, User
+from sentry.services.hybrid_cloud.user import APIUser
 
 
 class ActorSerializer(Serializer):
     def serialize(self, obj, attrs, user, **kwargs):
-        if isinstance(obj, User):
+        if isinstance(obj, User) or isinstance(obj, APIUser):
             actor_type = "user"
             name = obj.get_display_name()
```

Comment:
Would it make sense for `get_display_name` to simply become a `staticmethod` that is provided arguments it needs?  That prevent the need for this helper to be duplicated.

Reply:
Could be a good standalone function for the user service to expose as well.

Reply:
I can't, quite yet just reference `ApiUser` as there are some ActorSerializer callsites that do reference raw ORMs still.  This compromise remains until we can close that hole.